rules/common.smk

```diff
@@ -151,6 +151,20 @@ def dataset_version(
     return dataset
 
 
+def handle_data_requests(params, output):
```

Comment:
Can be removed: Only used in one location if I see it correctly, and there you can either integrate it or even simplify that particular rule.